packages/@lwc/compiler/src/options.ts

```diff
@@ -14,6 +14,7 @@ type RecursiveRequired<T> = {
 const DEFAULT_OPTIONS = {
     isExplicitImport: false,
     preserveHtmlComments: false,
+    disableStaticContentOptimization: false,
```

Comment:
Final comment, since the current default value is `false` what is the value of setting it again here again to `false`?

Reply:
My thought process was to mirror the default behavior of the template compiler but now that you mention it, this does seem redundant and could cause extra maintenance overhead in the future.  

I've removed it and will go with the default values on the template compiler.